js/main.js

```diff
@@ -0,0 +1,33 @@
+var photoUrl = document.querySelector('#photo-url');
+photoUrl.addEventListener('input', photoFunction);
+
+function photoFunction(event) {
+  var imageInput = document.querySelector('#photo-url').value;
+  document.querySelector('.image').src = imageInput;
+  return imageInput;
+}
+
+var journalEntry = document.querySelector('form');
+journalEntry.addEventListener('submit', submitEvent);
+
+function submitEvent(event) {
+  var newObj = {};
+  newObj.title = document.querySelector('#title-name').value;
+  newObj.img = document.querySelector('#photo-url').value;
+  newObj.notes = document.querySelector('#notes').value;
+  newObj.currentEntryId = data.nextEntryId;
+  data.nextEntryId++;
+
+  data.entries.unshift(newObj);
+
+  document.querySelector('.image').reset();
+  document.querySelector('#title-name').reset();
+  document.querySelector('#notes').reset();
```

Comment:
I would call reset on the form itself instead of calling it on the individual inputs.